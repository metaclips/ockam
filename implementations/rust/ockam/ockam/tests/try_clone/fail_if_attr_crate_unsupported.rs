use ockam_core::TryClone;

#[derive(TryClone)]
#[try_clone(crate = "my_crate")]
pub struct Tmp {
    a: u32,
}

fn main() {}
