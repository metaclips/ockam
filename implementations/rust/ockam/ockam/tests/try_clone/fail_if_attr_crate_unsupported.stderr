error: only `ockam`, `ockam_core` or `crate` are supported, got `my_crate`
 --> tests/try_clone/fail_if_attr_crate_unsupported.rs:4:21
  |
4 | #[try_clone(crate = "my_crate")]
  |                     ^^^^^^^^^^
